src/App.js

```diff
@@ -7,6 +8,11 @@ import { Footer } from "./components/Footer/Footer";
 import { Navigation } from "./components/Navigation/Navigation";
 import { VideoListing } from "./pages/video-listing/VideoListing";
 import { SingleVideoCard } from "./components/Card/SinglevideoCard/SingleVideoCard";
+import { LikedVideoPage } from "./pages/LikedVideoPage/LikedVideoPage";
+import { WatchLater } from "./pages/WatchlaterPage/WatchLater";
+import { HistoryPage } from "./pages/HistoryPage/HistoryPage";
+import { PlaylistPage } from "./pages/PlaylistPage/PlaylistPage";
+import { SinglePlaylistPage } from "./pages/SinglePlaylistPage/SinglePlaylistPage";
```

Comment:
You can make use of barrels (a common index file) to import these files to make the code concise.